CsvParser/csvParser.go

```diff
@@ -116,13 +115,15 @@ func convertDefaultIncidentToIncidentData(i incident.DefaultIncident) incident.I
 	return inc
 }
 
-//Takes in a location code and breaks it down into different location levels
-//Returns an array of string, where each string represents a location
+// Takes in a location code and slices it into different location levels
+// Returns an array of strings, where each string represents a location
 func parseLocationCode(locationCode string) []string {
 	locationCodesArr := make([]string, 0)
+	customLocationLen := 6 // Examples: nausca, nauscaclaremont (nonCustom) 
```

Comment:
Would it be accurate to name this variable stateLocationLen ? How is it custom vs non-custom?